manual/adobeshockwaveplayer/adobeshockwaveplayer.nuspec

```diff
@@ -1,29 +1,31 @@
-<?xml version="1.0"?>

```

Comment:
There is no need to remove these lines. Can you put them back in?

Reply:
This line was corrected by adding encoding following the standards of the chocolatey model package
`<?xml version="1.0"?>` replaced by `<?xml version="1.0" encoding="utf-8"?>`

Reply:
The line you changed didn't fix anything so there is no need to replace it.